components/member-card/showskills.js

```diff
@@ -0,0 +1,36 @@
+import { skills_data } from './skills';
+import classNames from './card.module.scss';
+import PropTypes from 'prop-types';
+
+const Showskills = ({ show }) => {
+  return (
+    <div className={classNames.showskills}>
```

Comment:
please use the classNames to be `camelCase`

so - `showSkills`

Reply:
done.